examples/plotting/spans_and_strips.py

```diff
@@ -1,3 +1,12 @@
+''' An example demonstration the use of `hspan`, `wspan`, and `vstrip`.
+It also uses the `HoverTool`.
```

Comment:
I'd suggest elaborating at least a little, maybe:
```suggestion
It also demonstrates how the default ``HoverTool`` interacts with these glyphs.
```